docs/how_to_guide/adding_a_coalignment_method.rst

```diff
@@ -0,0 +1,41 @@
+.. _sunkit-image-how-to-guide-adding-a-new-coalignment-method:
+
+****************************
+Add a New Coalignment Method
+****************************
+
+You can add a custom coalignment method in the sunkit-image package using the decorator ``sunkit_image.coalignment_module.util.decorators.register_coalignment_method``:
```

Comment:
These should be link able.

If they are not, then you will need to add files inside https://github.com/sunpy/sunkit-image/tree/main/docs/code_ref to enable that. Unfortunately, it's not automatic.

Reply:
I initially thought of referencing, but again since we would be replacing the current coalignment with the new one I thought it would make sense to avoid adding extra things and reference it later.

Reply:
We should get them working, the renaming process won't be difficult. 

Should just be a find and replace.